frontend/lib/Providers/EndpointListModel.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:math';
+
+import 'package:adpv_frontend/EndpointList/EnpointList.dart';
+import 'package:adpv_frontend/Models/EndpointData.dart';
+import 'package:adpv_frontend/Repository/AbstractEndpointRepository.dart';
+import 'package:adpv_frontend/Repository/RestClient.dart';
+import 'package:dio/dio.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'dart:developer' as developer;
+import '../Models/Endpoint.dart';
+import '../Models/EndpointSummary.dart';
+
+class ExpansionPanelEndpoint {
+  ExpansionPanelEndpoint({
+    required this.recentData,
+    required this.label,
+    required this.id,
+    required this.fields,
+    required this.buttonColor,
+  });
+
+  Map<String, dynamic> recentData;
+  String label;
+  int id;
+  List<String> fields;
+  Color buttonColor;
+
+  void setRecentData(EndpointData data) {
+    fields = data.getAllRecentFields();
+    recentData =
+        data.dataList[0].map((key, value) => MapEntry(key.toString(), value));
+    if (recentData != null) {
+      recentData.removeWhere((key, value) => key.toString() == 'timestamp');
+    }
```

Comment:
Ten `if` chyba tutaj nie będzie potrzebny, bo `recentData` nigdy nie będzie `null`'em. Co najwyżej mapa może być pusta, ale to nie powinno wpłynąć na działanie `removeWhere`

Reply:
Bez tego ifa mi wypierdalało błąd ale mg jeszcze to przejrzeć